x/ibc/02-client/handler_test.go

```diff
@@ -0,0 +1,53 @@
+package client_test
+
+import (
+	client "github.com/cosmos/cosmos-sdk/x/ibc/02-client"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	localhosttypes "github.com/cosmos/cosmos-sdk/x/ibc/09-localhost/types"
+)
+
+func (suite *ClientTestSuite) TestHandleCreateClientLocalHost() {
```

Comment:
it would have been nice to note that since `suite.SetupTest()` is not being ran, test case state changes are persisted through each test meaning test case ordering matters. This could be confusing for later developers who modify this test or copy its structure.